alembic/versions/1e2359bf1fbc_adds_requirement_and_mapping_options_.py

```diff
@@ -0,0 +1,71 @@
+    op.create_table('strategy_requirement',
+    sa.Column('id', postgresql.UUID(as_uuid=True), nullable=False),
+    sa.Column('project_id', postgresql.UUID(as_uuid=True), nullable=True),
+    sa.Column('strategy_id', postgresql.UUID(as_uuid=True), nullable=True),
+    sa.Column('created_by', postgresql.UUID(as_uuid=True), nullable=True),
+    sa.Column('created_at', sa.DateTime(), nullable=True),
+    sa.Column('field', sa.String(), nullable=True),
+    sa.Column('is_input', sa.Boolean(), nullable=True),
+    sa.Column('description', sa.String(), nullable=True),
+    sa.ForeignKeyConstraint(['created_by'], ['user.id'], ondelete='SET NULL'),
+    sa.ForeignKeyConstraint(['project_id'], ['cognition.project.id'], ondelete='CASCADE'),
+    sa.ForeignKeyConstraint(['strategy_id'], ['cognition.strategy.id'], ondelete='CASCADE'),
+    sa.PrimaryKeyConstraint('id'),
+    schema='cognition'
+    )
+    op.create_index(op.f('ix_cognition_strategy_requirement_created_by'), 'strategy_requirement', ['created_by'], unique=False, schema='cognition')
+    op.create_index(op.f('ix_cognition_strategy_requirement_project_id'), 'strategy_requirement', ['project_id'], unique=False, schema='cognition')
+    op.create_index(op.f('ix_cognition_strategy_requirement_strategy_id'), 'strategy_requirement', ['strategy_id'], unique=False, schema='cognition')
+
+    op.create_table('strategy_requirement_mapping_option',
+    sa.Column('id', postgresql.UUID(as_uuid=True), nullable=False),
+    sa.Column('project_id', postgresql.UUID(as_uuid=True), nullable=True),
+    sa.Column('strategy_requirement_id', postgresql.UUID(as_uuid=True), nullable=True),
+    sa.Column('created_by', postgresql.UUID(as_uuid=True), nullable=True),
+    sa.Column('created_at', sa.DateTime(), nullable=True),
+    sa.Column('value', sa.String(), nullable=True),
+    sa.ForeignKeyConstraint(['created_by'], ['user.id'], ondelete='SET NULL'),
+    sa.ForeignKeyConstraint(['project_id'], ['cognition.project.id'], ondelete='CASCADE'),
+    sa.ForeignKeyConstraint(['strategy_requirement_id'], ['cognition.strategy_requirement.id'], ondelete='CASCADE'),
+    sa.PrimaryKeyConstraint('id'),
+    schema='cognition'
+    )
+    op.create_index(op.f('ix_cognition_strategy_requirement_mapping_option_created_by'), 'strategy_requirement_mapping_option', ['created_by'], unique=False, schema='cognition')
+    op.create_index(op.f('ix_cognition_strategy_requirement_mapping_option_project_id'), 'strategy_requirement_mapping_option', ['project_id'], unique=False, schema='cognition')
+    op.create_index(op.f('ix_cognition_strategy_requirement_mapping_option_strategy_requirement_id'), 'strategy_requirement_mapping_option', ['strategy_requirement_id'], unique=False, schema='cognition')
+    # ### end Alembic commands ###
+
+
+
+def downgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.drop_table('strategy_requirement', schema='cognition')
```

Comment:
`strategy_requirement` must be dropped after `strategy_requirement_mapping_options` due to foreign key constraints